SignalMessaging/contacts/OWSContactsSyncing.m

```diff
@@ -139,9 +139,9 @@ - (void)sendSyncContactsMessageIfNecessary
             success:^{
                 DDLogInfo(@"%@ Successfully sent contacts sync message.", self.logTag);
 
-                [[TSStorageManager sharedManager] setObject:messageData
-                                                     forKey:kTSStorageManagerOWSContactsSyncingLastMessageKey
-                                               inCollection:kTSStorageManagerOWSContactsSyncingCollection];
+                [TSStorageManager.dbReadWriteConnection setObject:messageData
```

Comment:
We want to remove usage of the storage managers "global" connections.  A first step towards that is removing the convenience methods from `TSStorageManager` and making usage of these connections explicit.